packages/components/src/alignment-matrix-control/types.ts

```diff
@@ -0,0 +1,50 @@
+export type AlignmentMatrixControlValue =
+	| 'top left'
+	| 'top center'
+	| 'top right'
+	| 'center left'
+	| 'center'
+	| 'center center'
+	| 'center right'
+	| 'bottom left'
+	| 'bottom center'
+	| 'bottom right';
+
+export type AlignmentMatrixControlProps = {
+	/**
+	 * Accessible label. If provided, sets the `aria-label` attribute of the
+	 * underlying <Composite/> component.
```

Comment:
While we're at it, I'm finding these references to the `Composite` component a bit strange because it's exposing an implementation detail in a way that isn't even helpful. (Nobody knows what the Composite component is 😆)

Since this one is about accessibility, I think I would prefer something more like this that describes it in terms of web standards:

```suggestion
	 * Accessible label. If provided, sets the `aria-label` attribute of the
	 * underlying `grid` widget.
```

If that's too much, I'm also fine with a more generic description like "Accessible label for the control".

Reply:
Cool! I've applied the change and also carried it over to the README, which is where the reference to `Composite` originally came from. There were three references there (the third was for `className`) and they've all been replaced in [3f54beb](https://github.com/WordPress/gutenberg/pull/46162/commits/3f54bebffc7a29d32e4d4d912b7e2a8b48b2e14d)